ArmaforcesMissionBot/Handlers/LoadupHandler.cs

```diff
@@ -146,8 +146,7 @@ private async Task LoadMissions(SocketGuild guild)
 					{
 						mission.Title = embed.Title;
                         mission.Description = embed.Description;
-                        var user = embed.Author.Value.Name.Split("#");
-                        mission.Owner = _client.GetUser(user[0], user[1]).Id;
+                        mission.Owner = ulong.Parse(embed.Author.Value.Url.Substring(26));
```

Comment:
It cannot stay like this.

1) It's magic number. I know it is from the length of `https://discord.com/users/` string, but it's unacceptable way of using it. You should create some eg. `DiscordConstants` class and put the base url (mentioned before) there. Then you can use `DiscordConstants.UserUrlBase.Length` or something instead of `26`.

2) It would be nice if you could add screenshot to PR description how it looks like with additional user url. I will re-review after getting a screenshot.